crypto/hashers.go

```diff
@@ -1,40 +0,0 @@
-// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
This delete is appropriate because I've moved the relevant hash enums into `DigitallySigned` 
The replacement lookup dictionaries are in `signature.go` 

`HashingStrategy` is entirely different.